samples/matter/lock/README.rst

```diff
@@ -377,6 +377,35 @@ To test switching between Thread and Wi-Fi, complete the following steps:
 
       D: 823 [DL]WiFiManager has been initialized
 
+Device firmware upgrade over Matter with support for switching between Thread and Wi-Fi
+==================================
```

Comment:
This must be as long as the previous line, I believe.

Reply:
This already fixed.